### DISCUSSION TITLE
EqualStreetNames.Hannover

### DISCUSSION THREAD

**Discussion by nortix** [EqualStreetNames/equalstreetnames#765] [New city]
Hi,
i've finally added all Wikidata IDs to the streets of [Hannover, Germany](https://github.com/nortix/equalstreetnames-hannover), and i want to transfer the repository to the EqualStreetNames organization. Could you add me to the organization to finish the last step (MAPBOX_TOKEN). 
Thank you very much.

**Comment by jbelien** [comment]
Hello @nortix , you should have received an invite to join @EqualStreetNames/hannover ; once you've joined you'll be able to transfer your repository to EqualStreetNames organization.

You can already add the `MAPBOX_TOKEN` secret before doing the transfer.

**Comment by nortix** [comment]
Done. Hope i got the MAPBOX_TOKEN right. Thanks for reaching out so fast.

**Comment by jbelien** [comment]
The token doesn't seem to be set, please add it here: https://github.com/EqualStreetNames/equalstreetnames-hannover/settings/secrets/actions/new

I'll take care of the last steps in the next few days.

**Comment by nortix** [comment]
Sorry to bother you. I have no idea how to solve the token issue and the URL you mentioned leads nowhere (404).

**Comment by jbelien** [comment]
> Sorry to bother you. I have no idea how to solve the token issue and the URL you mentioned leads nowhere (404).

Sorry, I indeed didn't give you the correct rights. It's now fixed, you should be able to add the token using the link from my previous comment.

**Comment by nortix** [comment]
@jbelien I did it. Can you please check it? Thank you.

**Comment by jbelien** [comment]
Everything is now merged.
I'll finish the deployment this weekend.

**Comment by jbelien** [comment]
🚀 Deployed: https://hannover.equalstreetnames.eu/

Great work! 👍 

> **Note**
> I just had to fix a `wikidata` tag on "Berta-Weiß-Weg"
> See https://www.openstreetmap.org/changeset/135764626
